src/map/clif.c

```diff
@@ -18155,21 +18155,30 @@ static void clif_parse_Adopt_reply(int fd, struct map_session_data *sd)
 ///     1 = Boss is alive (position update) (BOSS_INFO_ALIVE).
 ///     2 = Boss is alive (initial announce) (BOSS_INFO_ALIVE_WITHMSG).
 ///     3 = Boss is dead (BOSS_INFO_DEAD).
-static void clif_bossmapinfo(int fd, struct mob_data *md, short flag)
+static void clif_bossmapinfo(struct map_session_data *sd, struct mob_data *md, short flag)
 {
+	int fd = sd->fd;
+
 	WFIFOHEAD(fd,70);
 	memset(WFIFOP(fd,0),0,70);
 	WFIFOW(fd,0) = 0x293;
 
-	if( md != NULL ) {
-		if( md->bl.prev != NULL ) { // Boss on This Map
-			if( flag ) {
-				WFIFOB(fd,2) = 1;
-				WFIFOL(fd,3) = md->bl.x;
-				WFIFOL(fd,7) = md->bl.y;
-			} else
-				WFIFOB(fd,2) = 2; // First Time
-		} else if (md->spawn_timer != INVALID_TIMER) { // Boss is Dead
+	switch (flag) {
+		case 0:
+			WFIFOB(fd,2) = 0;
```

Comment:
in new/changed code, after ``,`` should be space ``, ``

Reply:
Additionally the indentation is wrong.  The cases should have the same indentation level as the `switch` statement.